src/codegen/spirv.zig

```diff
@@ -531,7 +531,17 @@ const DeclGen = struct {
             },
             .Enum => return self.arithmeticTypeInfo(ty.intTagType(mod)),
             // As of yet, there is no vector support in the self-hosted compiler.
-            .Vector => self.todo("implement arithmeticTypeInfo for Vector", .{}),
+            .Vector => blk: {
+                const child_type = ty.childType(mod);
+                const child_ty_info = try self.arithmeticTypeInfo(child_type);
+                break :blk ArithmeticTypeInfo{
+                    .bits = @intCast(child_ty_info.bits * ty.vectorLen(mod)),
+                    .backing_bits = @intCast(child_ty_info.backing_bits * ty.vectorLen(mod)),
```

Comment:
This is used in some code to determine the size of the result time. I think these should represent scalars, so they should be inherited from the child_ty_info